manifests/vhost.pp

```diff
@@ -2296,7 +2297,7 @@
   }
   # NOTE(pabelanger): This code is duplicated in ::apache::vhost::custom and
```

Comment:
As indicated by this comment, this code is duplicated in https://github.com/puppetlabs/puppetlabs-apache/blob/c3fc5a74019ead9869194c33c8577ce06441c52f/manifests/vhost/custom.pp and one might want to consider replicating the changes there.